sources/tools/Stride.Importer.Assimp/MeshConverter.cs

```diff
@@ -250,24 +250,32 @@ private unsafe Rendering.Skeleton ProcessSkeleton(Scene* scene)
             };
         }
 
-        private unsafe AnimationInfo ProcessAnimations(Scene* scene)
+        private unsafe AnimationInfo ProcessAnimations(Scene* scene, int animationIndex)
         {
             var animationData = new AnimationInfo();
             var visitedNodeNames = new HashSet<string>();
 
-            if (scene->MNumAnimations > 1)
-                Logger.Warning($"There are {scene->MNumAnimations} animations in this file, using only the first one.");
-
             var nodeNames = new Dictionary<IntPtr, string>();
             GenerateNodeNames(scene, nodeNames);
 
             // register the nodes and fill hierarchy
             var meshIndexToNodeIndex = new Dictionary<int, List<int>>();
             RegisterNodes(scene->MRootNode, -1, nodeNames, meshIndexToNodeIndex);
 
-            for (uint i = 0; i < Math.Min(1, scene->MNumAnimations); ++i)
```

Comment:
Was this supporting multiple animations before? Are we not reducing functionality then?

Reply:
That's the min of 1 and the animation count, so it was a weird way of saying skip that code if there's zero animations. Maybe at one point it was kind of supported and then the support was removed? Not sure why else they'd write it that weirdly.